de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterWindowEventHandler.java

```diff
@@ -353,7 +362,20 @@ private void init(EventBus eventBus) {
     }
 
     @Override
-    public void createNewApp(UseInNewAppEvent event) {
-        presenter.show(ConfigFactory.appsIntegrationWindowConfig(null), true);
+    public void useToolInNewApp(UseToolInNewAppEvent event) {
+        AppsIntegrationWindowConfig aiwc = ConfigFactory.appsIntegrationWindowConfig(null);
```

Comment:
It looks like this code is duplicated in `AppEditorWindow`.  I wonder if it would be better to consolidate the code template creation there?